modules/core/include/opencv2/core/hal/intrin.hpp

```diff
@@ -329,11 +329,17 @@ CV_CPU_OPTIMIZATION_HAL_NAMESPACE_END
 
 #endif
 
+#if CV_AVX2
+
+#include "opencv2/core/hal/intrin_avx2.hpp"
+
+#endif
+
 //! @addtogroup core_hal_intrin
 //! @{
 
 #ifndef CV_SIMD128
-//! Set to 1 if current compiler supports vector extensions (NEON or SSE is enabled)
+//! Set to 1 if current compiler supports vector extensions (NEON or SSE or VSX is enabled)
```

Comment:
AVX?

Reply:
Just update doc for [Power VSX](https://github.com/opencv/opencv/blob/master/modules/core/include/opencv2/core/hal/intrin_vsx.hpp). Please consider this branch isn't ready for review yet, I made so many changes and still working on other things. I would expect to finish it in a week or two.